DivvyUp.sol

```diff
@@ -370,31 +330,31 @@ contract DivvyUp is ERC20Interface, Owned, DivvyUpInterface {
     /**
      * Converts all incoming counter to tokens for the caller
      */
-    function purchaseTokensERC20WithReferrer(uint256 amount, address referrer)
+    function purchaseTokensERC20WithReferrer(uint256 amount, address)
```

Comment:
This function and the one above need to be merged to be one function which does't take an address.